packages/graphql-transformers-e2e-tests/src/CloudFormationClient.ts

```diff
@@ -110,31 +105,30 @@ export class CloudFormationClient {
       'UPDATE_ROLLBACK_COMPLETE_CLEANUP_IN_PROGRESS',
       'UPDATE_ROLLBACK_IN_PROGRESS',
     ],
-    maxPolls: number = 1000,
-    pollInterval: number = 20,
+    maxPolls = 1000,
+    pollInterval = 20,
   ): Promise<CloudFormation.Stack> {
     const stack = await this.describeStack(name);
     if (success.includes(stack.StackStatus)) {
       return Promise.resolve(stack);
-    } else if (failure.includes(stack.StackStatus)) {
+    } if (failure.includes(stack.StackStatus)) {
```

Comment:
Is changing `else if` to `if` intentional here?

Reply:
this is all auto lint changes

Reply:
```suggestion
    }
    if (failure.includes(stack.StackStatus)) {
```

Reply:
Probably we need to check why lint is forcing this change. Earlier if the first condition passes, it would skip the remaining checks. Now it executes all the checks one by one.